### ISSUE TITLE
WebId search is not working in the twitter social app

### ISSUE THREAD

**Issue by hitman401** [joshuef/webIdManager#10]
Looks like it is fetching, but throws an error popup window stating the WebId is not found.

```
Target WebID doc: [{"@id":"safe://sp.random","http://purl.org/dc/terms/title":[{"@value":"hghghg's profile document"}],"@type":["http://xmlns.com/foaf/0.1/PersonalProfileDocument"],"http://xmlns.com/foaf/0.1/maker":[{"@id":"safe://sp.random#me"}],"http://xmlns.com/foaf/0.1/primaryTopic":[{"@id":"safe://sp.random#me"}]},{"@id":"safe://sp.random#me","@type":["http://xmlns.com/foaf/0.1/Person"],"http://xmlns.com/foaf/0.1/name":[{"@value":"hghghg"}],"http://xmlns.com/foaf/0.1/nick":[{"@value":"sdsd@#6767<."}]},{"@id":"safe://sp.random/posts","http://purl.org/dc/terms/title":[{"@value":"Container for social apps posts"}],"http://safenetwork.org/safevocab/typeTag":[{"@value":"303030"}],"http://safenetwork.org/safevocab/xorName":[{"@value":"193,204,24,218,166,124,227,164,50,241,5,115,68,4,194,96,79,211,54,192,70,77,22,115,36,159,182,13,116,107,39,7"}],"@type":["http://safenetwork.org/safevocab/Posts"]}]
Header.jsx:19 WebID entered not found: TypeError: Cannot read property 'object' of undefined
    at safe://tweet.krishna/src.ab486079.js:718:4359
    at L (safe://tweet.krishna/src.ab486079.js:447:4401)
    at Generator._invoke (safe://tweet.krishna/src.ab486079.js:447:4189)
    at Generator.t.(anonymous function) [as next] (safe://tweet.krishna/src.ab486079.js:447:4580)
    at r (safe://tweet.krishna/src.ab486079.js:718:619)
    at safe://tweet.krishna/src.ab486079.js:718:713
    at <anonymous>
```

**Comment by lionel-faber** [comment]
I have faced the same issue. I've noticed that the error occurs when searching for a profile that does not have a picture uploaded when creating a webId. I've uploaded the mock vault file (added the .txt extension to support upload) account secret and password "joseph lionel faber"
[MockVault.txt](https://github.com/joshuef/webIdManager/files/2208823/MockVault.txt)

**Comment by lionel-faber** [comment]
NOTE: If the profile is edited and an image is added to the web ID. Then the profile is fetched and the page is updated

**Comment by bochaco** [comment]
I suspect this is not with the missing image but with the missing website like reported in this issue: https://github.com/joshuef/eat/issues/1, which was already fixed. Missing image was being handled before this fix so it shouldn't be an issue.
In order to reproduce, can you please confirm what's the webid you created in the mock file you shared @lionel1704 ?

**Comment by lionel-faber** [comment]
Yep! There are 4 web IDs in the mock file
with picture:
- safe://mywebid.lionel
- safe://webid.faber
without picture:
- safe://webid.joseph
- safe://webid.nopic

**Comment by bochaco** [comment]
I just tried loading them all with the social app and they are loaded perfectly. Can you please verify with with the mock file you shared if you can reproduce it and indicate with which webid exactly you are reproducing it? since I'm not seeing any error with any of those webids @lionel1704

**Comment by bochaco** [comment]
Ok, @lionel1704 , I was testing with latest version which incudes the fix I mentioned above for joshuef/eat#1. So I was able to reproduce it with safe://webid.joseph, it's fixed now. Thanks a lot!!
